clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -722,6 +709,12 @@ struct MCDCCoverageBuilder {
       return I->second;
   }
 
+  /// Return the LHS Decision ({0,0} if not set).
+  const DecisionIDPair &back() {
+    assert(DecisionStack.size() >= 1);
```

Comment:
This assert should be unnecessary. `SmallVector` already asserts for you:

```
  reference back() {
    assert(!empty());
    return end()[-1];
  }
  const_reference back() const {
    assert(!empty());
    return end()[-1];
  }
```

Reply:
Dropped in the latest change. Thanks for pointing this out.